src/Cynthia.Card/src/Cynthia.Card.Common/CardEffects/DIY/Skellige/Gold/Crowmother.cs

```diff
@@ -1,47 +1,47 @@
-using System.Linq;
-using System.Threading.Tasks;
-using Alsein.Extensions;
+// using System.Linq;
+// using System.Threading.Tasks;
+// using Alsein.Extensions;
 
-namespace Cynthia.Card
-{
-    [CardEffectId("70159")]//
-    public class Crowmother : CardEffect, IHandlesEvent<AfterCardDiscard>
-    {//
-        public Crowmother(GameCard card) : base(card) { }
```

Comment:
Looks like we commented out all the codes. Did you test it? I am a bit worried about it might break the game when bringing this card or play this card